framework_tvb/tvb/core/services/import_service.py

```diff
@@ -416,10 +450,19 @@ def load_datatype_from_file(self, current_file, op_id, datatype_group=None, curr
             h5_file = BurstConfigurationH5(current_file)
             burst = BurstConfiguration(current_project_id)
             burst.fk_simulation = op_id
-            # burst.fk_operation_group = TODO
-            # burst.fk_metric_operation_group = TODO
             h5_file.load_into(burst)
             result = burst
+        elif h5_class is OperationGroupH5:
+            h5_file = OperationGroupH5(current_file)
+            operation_group = OperationGroup(current_project_id)
+            h5_file.load_into(operation_group)
+            result = operation_group
+
+        elif h5_class is DatatypeMeasureH5:
+            h5_file = DatatypeMeasureH5(current_file)
```

Comment:
The way you are handling the datatype measure entities looks redundant to me. 
Either you keep this `if` and load directly the `DatatypeMeasureIndex` from the `DatatypeMeasureH5`, or you define the new `DatatypeMeasure` object and go through the generic flow of loading from H5 that you do in the `else` below: 
`datatype, generic_attributes = h5.load_with_links(current_file)
`

Reply:
Seeing the code here and knowing that this part is problematic also for the data migration task, I tend to vote for adding the DatatypeMeasure object, so we can use the generic code to go from DatatypeMeasureH5 to DatatypeMeasureIndex and avoid the need to always treat it as a special case.